frontend/src/component/admin/apiToken/ApiTokenIcon/ApiTokenIcon.tsx

```diff
@@ -11,10 +11,30 @@ interface IApiTokenIconProps {
     secret?: string;
 }
 
-export const ApiTokenIcon: FC<IApiTokenIconProps> = ({ secret }) => {
+export const ApiTokenIcon: FC<IApiTokenIconProps> = ({
+    secret,
+    project,
+    projects,
+}) => {
     const tokenFormat = secret?.includes(':') ? 'v2' : 'v1'; // see https://docs.getunleash.io/reference/api-tokens-and-client-keys#format
     const isWildcardToken = secret?.startsWith('*:');
-    const isOrphanedToken = tokenFormat === 'v2' && !isWildcardToken;
+    const hasProjects = useMemo(() => {
+        if (
+            projects &&
+            Array.isArray(projects) &&
+            projects.length > 1 &&
```

Comment:
what about the case where you have `projects: ['a']`? That's a potential pitfall here, isn't it? Shouldn't this be `projects.length >= 1`?